### DISCUSSION TITLE
Is this malware?

### DISCUSSION THREAD

**Discussion by gavinsteinhoff** [dimaguy/ImpostorHQ#14] [General]
I started it up, but it asked for my cc info

**Comment by XtraCube** [comment]
Lol no. If it asked for your cc info, then you downloaded it from somewhere other than Github. Try downloading from here: https://github.com/dimaguy/ImpostorHQ/releases
(Also if this is a joke, you made me laugh)

**Comment by dimaguy** [comment]
Hi, no we do not collect any data.
You might be under attack from a network of terrorists attacking you for the sole purpose of getting your cc info.
***/s***
